orc/src/main/java/org/apache/iceberg/orc/ORC.java

```diff
@@ -191,28 +198,26 @@ private Context(long stripeSize, long blockSize) {
         this.blockSize = blockSize;
       }
 
-      static Context dataContext(Configuration conf) {
-        long stripeSize = TableProperties.ORC_STRIPE_SIZE_BYTES_DEFAULT;
-        stripeSize = conf.getLong(OrcConf.STRIPE_SIZE.getAttribute(), stripeSize);
-        stripeSize = conf.getLong(OrcConf.STRIPE_SIZE.getHiveConfName(), stripeSize);
-        stripeSize = conf.getLong(TableProperties.ORC_STRIPE_SIZE_BYTES, stripeSize);
+      static Context dataContext(Map<String, String> conf) {
```

Comment:
Could we always use `config`  to indicate that we are referring a configure `Map<String, String>` ?  The `conf` looks like a hadoop configuration in my mind.